doc/emergency-banner.md

```diff
@@ -0,0 +1,19 @@
+# Emergency banner
+
+See the [opsmanual](https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#adding-emergency-publishing-banners) for information about what the Emergency Banner is and when it should be deployed.
+
+## Running the Emergency Banner locally
+
+### Deploying the Emergency banner
+
+```
+bundle exec rake emergency_banner:deploy['{campaign_class}','{heading}','{short_description}','{link}']
+```
+
```

Comment:
@nickcolley perhaps that would come into this card https://trello.com/c/3pKoTsFM/49-review-the-frontend-code-and-markup ?